edxval/api.py

```diff
@@ -879,7 +891,7 @@ def import_from_xml(xml, edx_video_id, course_id=None):
 
         return
     except ValidationError as err:
-        logger.exception(err.message)
+        logger.exception(err)
```

Comment:
I think we don't need to pass the `err` to `logger.exception` because `logger.exception` is already logging the exception message.